dynamic.js

```diff
@@ -70,12 +77,30 @@ module.exports = function (fastify, opts, next) {
     if (basePath) swaggerObject.basePath = basePath
     if (consumes) swaggerObject.consumes = consumes
     if (produces) swaggerObject.produces = produces
-    if (definitions) swaggerObject.definitions = definitions
-    if (securityDefinitions) {
-      swaggerObject.securityDefinitions = securityDefinitions
-    }
-    if (security) {
-      swaggerObject.security = security
+    if (securityDefinitions) swaggerObject.securityDefinitions = securityDefinitions
+    if (security) swaggerObject.security = security
+    if (components) swaggerObject.components = components
+    if (servers) swaggerObject.servers = servers
+
+    if (opts && opts.addFastifySchemas) {
+      // extract definitions from fastify schemas
+      const schemas = fastify._schemas.store
```

Comment:
I would prefer if you did not access this here. Can you send a PR to fastify to get an iterator over the schema store? Or a  `getAllSchemas()` method or accessor?

Reply:
Good point. I've thought about this. For now it will remain just for tests and after PR to fastify I'll replace it with appropriate method.